net/host.go

```diff
@@ -61,16 +66,17 @@ type Host struct {
 // NewHost returns a new Host.
 // The host implemented in this package is swap-specific; ie. it supports swap-specific
 // messages (initiate and query).
-func NewHost(cfg *p2pnet.Config) (*Host, error) {
+func NewHost(cfg *p2pnet.Config, isRelayer bool) (*Host, error) {
```

Comment:
This was a little awkward. The config passed in is for the "base class" from another repo, so I couldn't add a new field.

Reply:
you could make a config for this `Host` that accepts the `cfg` and `isRelayer`?

Reply:
You can only inherit methods from an anonymous struct field. Access to nested fields still require the type name prefix, so composing wasn't a good option. I just cloned the field's in `p2pnet`'s `Config` into our net's `Config`. More code to maintain another `Config`, but if our `net` is wrapping `p2pnet`'s `Host`, it should be wrapping the `Config` too in order to avoid transitive import requirements.